app/assets/stylesheets/_tables.scss

```diff
@@ -0,0 +1,36 @@
+[aria-sort] > * {
+  position: relative;
+
+  &:after {
+    font-size: em(10, 19);
```

Comment:
GOV.UK uses pixels for font sizes everywhere. (I personally prefer `em` as well but it makes sense to stay consistent.) When you use [the typography mixins from the frontend toolkit](https://github.com/alphagov/govuk_frontend_toolkit/blob/master/stylesheets/_typography.scss) the sizes will automatically be in `px`.

In this case I'd actually prefer a bigger arrow, but maybe double check with the design community first before changing that.